workflow-manager/main.go

```diff
@@ -93,25 +181,66 @@ func main() {
 		log.Fatal(err)
 	}
 
-	var readyBatches []string
-	switch service {
+	var readyIngestionBatches []string
+	switch ingestorService {
+	case "s3":
+		readyIngestionBatches, err = getReadyIngestionBatchesS3(context.Background(), ingestorInputBucket, *ingestorIdentity)
+		if err != nil {
+			log.Fatal(err)
+		}
+	case "gs":
+		readyIngestionBatches, err = getReadyIngestionBatchesGS(context.Background(), ingestorInputBucket)
+		if err != nil {
+			log.Fatal(err)
+		}
+	default:
+		log.Fatalf("unknown service %s", ingestorService)
+	}
+
+	if err := launchIntake(context.Background(), readyIngestionBatches, ageLimit); err != nil {
+		log.Fatal(err)
+	}
+
+	var readyOwnValidationBatches []string
+	switch ownValidationService {
 	case "s3":
-		readyBatches, err = getReadyBatchesS3(context.Background(), inputBucket, *ingestorIdentity)
+		readyOwnValidationBatches, err = getReadyValidationBatchesS3(context.Background(),
+			ownValidationInputBucket, parseIsFirst(*isFirst), readyIngestionBatches, *ingestorIdentity)
+	case "gs":
+		readyOwnValidationBatches, err = getReadyValidationBatchesGS(context.Background(),
+			ownValidationInputBucket, parseIsFirst(*isFirst), readyIngestionBatches)
 		if err != nil {
 			log.Fatal(err)
 		}
+	default:
+		log.Fatalf("unknown service %s", ownValidationService)
+	}
+
+	log.Printf("found ready own validations: %v", readyOwnValidationBatches)
```

Comment:
This is going to be quite noisy and hard to read, with everything on one line. Let's just print the count of batches that we're going to work on. Each batch will print something as it's kicked off.